bindata/deployment/frr/metallb-frr.yaml

```diff
@@ -449,6 +505,35 @@ spec:
             readOnlyRootFilesystem: true
           command:
             - /controller
+        {{ if .IsOpenShift }}
+        - name: kube-rbac-proxy
```

Comment:
Can we have this section (which looks identical to the previous minus the port) in a separate file that we inject into the template?

Reply:
Not sure I follow, it already is? (`kube-rbac-controller.json`)

Reply:
Yep I realized it later on, we can change it once we remove the jq hell and get rid of the automatic generation (which is not so automatic anymore as a lot of content is expressed via jq overrides).